### PR TITLE
publishing agent status on topic

### PR DESCRIPTION
**Pull request by charvigupta96**
battery lvl, ip address, status, feedback freq

### CODE REVIEW

File: include/robot_agent.hpp [hunk 1]
```diff
@@ -35,6 +39,8 @@ class RobotAgent
     const int control_port_;
     const int feedback_freq_hz_;
```
Comment (rachelzh9) [inline include/robot_agent.hpp:42]: Did you add `feedback_freq_hz_` to the config file?

File: src/robot_agent.cpp [hunk 2]
```diff
@@ -127,6 +132,35 @@ void RobotAgent::timerCallback(const ros::TimerEvent& timer_event) {
...
+/**
+ * @brief Updates the agent status at feedback frequency
+ *
+ * @return void
+ *
+ */
+void RobotAgent::updateAgentStatus(const ros::TimerEvent& timer_event) {
+    robosar_messages::agents_status agent_status_;
+    agent_status_.ip_adress = ip_address_;
+    agent_status_.battery_lvl = bridgePtr->getBatteryLvl(); // hardcoded for testing, will change to jae's implementation
```
Comment (rachelzh9) [inline src/robot_agent.cpp]: This is using Jae's updates right?

**Review by rachelzh9 — CHANGES_REQUESTED** [review]

File: src/robot_agent.cpp (hunk 2 above)
Comment (charvigupta96) [reply src/robot_agent.cpp]: yes, removed comment

File: include/robot_agent.hpp (hunk 1 above)
Comment (charvigupta96) [reply include/robot_agent.hpp:42]: added freq_calculation_duration to the config file

**Review by rachelzh9 — CHANGES_REQUESTED** [review]
Are we able to change it such that we publish the agent info as lists on a single topic?

**Comment by jsongCMU** [comment]
Tested; looks good. Ready for merge after PR #39 is approved; otherwise get failure and not obvious why (duration = 0 by default, and this throws a divide by zero runtime error)

**Review by jsongCMU — APPROVED** [review]

**Pull request merged**